pyclaragenomics/bin/evaluate_paf

```diff
@@ -53,7 +55,7 @@ def evaluate_paf(truth_paf_filepath, test_paf_filepath, pos_tolerance=400, skip_
                 (truth_overlap.query_sequence_name == truth_overlap.target_sequence_name):
             continue
 
-        key = truth_overlap.query_sequence_name + truth_overlap.target_sequence_name
+        key = truth_overlap.query_sequence_name + "_" + truth_overlap.target_sequence_name
```

Comment:
Can you move the key generation to a function that appends name1 and name1 with an underscore? Should have done it last time to avoid this bug in the first place